Competition2017/robot-program-build.properties

```diff
@@ -23,12 +23,13 @@ guice_assisted.jar=${commonlib.libs}/guice-3.0/guice-assistedinject-3.0.jar
 inject.jar=${commonlib.libs}/guice-3.0/javax.inject.jar
 log4j.jar=${commonlib.libs}/log4j/log4j-1.2.17.jar
 commonsio.jar=${commonlib.libs}/commons-io-2.4/commons-io-2.4.jar
-derby.jar=${commonlib.libs}/derby/derby.jar
+derby.jar=${commonlib.libs}/Derby/derby.jar
 junit.jars=${commonlib.libs}/hamcrest-core-1.3.jar:${commonlib.libs}/junit-4.12.jar
 opencv.jar=${commonlib.libs}/opencv.jar
 ctre.jar=${commonlib.libs}/CTRLib.jar
+influxdb.jars=lib/influxdb-java-2.5.jar:lib/assertj-core-2.6.0.jar:lib/byte-buddy-1.6.5.jar:lib/byte-buddy-agent-1.6.5.jar:lib/converter-moshi-2.1.0.jar:lib/guava-20.0.jar:lib/hamcrest-core-1.3.jar:lib/junit-4.12.jar:lib/logging-interceptor-3.6.0.jar:lib/mockito-core-2.6.8.jar:lib/moshi-1.2.0.jar:lib/objenesis-2.5.jar:lib/okhttp-3.6.0.jar:lib/okio-1.11.0.jar:lib/retrofit-2.1.0.jar:lib/slf4j-api-1.7.23.jar:lib/slf4j-simple-1.7.23.jar
```

Comment:
Why do we need this many dependencies? It's a major red flag that something larger is going wrong.